### PR TITLE
40 add scenarios for standard amrs

### LINKED ISSUE #40: Add scenarios for standard AMRs
Add scenarios to test and simulate standard AMRs in `data/models`

### PR DESCRIPTION
**Pull request by sabinala**
This PR adds simulate and calibrate scenarios using both pyciemss and sciml for each of the new model AMRs in `data/models`. (Will add optimization scenarios asap.)

Closes #40

### CODE REVIEW

File: tests/report.py
Comment (fivegrant) [inline tests/report.py:77]: This has been merged into main with your other PR so this branch can be rebased in a way where this change no longer shows up.

File: scenarios/seir-stockflow/sciml/calibrate.json
Comment (fivegrant) [inline scenarios/seir-stockflow/sciml/calibrate.json:1]: I think we should hold off on adding `*/sciml/calibrate.json`s to other scenarios. I think the base the original one doesn't work yet and the payloads may have to be a bit different. tl;dr: remove `scenarios/*/sciml/calibrate.json` in this PR?

File: scenarios/seir-stockflow/pyciemss/optimize-calibrate.json
Comment (fivegrant) [inline scenarios/seir-stockflow/pyciemss/optimize-calibrate.json:16]: Is this name supposed to be different from the one in `optimize-simulation.json`?

**Review by fivegrant — CHANGES_REQUESTED** [review]
This PR is what we've been needing! I've pointed out some things to fix before merging.

Also, is it okay if you post the results JSON you're getting OR a screenshot from the dashboard so we know what is succeeding or failing.

File: scenarios/seir-stockflow/sciml/calibrate.json
Comment (jClugstor) [reply scenarios/seir-stockflow/sciml/calibrate.json:1]: I think the reason that the calibrate fails for the sirhd scenario is a weird XML parsing bug. It fails to parse a part of the AMR for whatever reason. Calibrate works on other models.

Comment (jClugstor) [reply scenarios/seir-stockflow/sciml/calibrate.json:1]: Specifically, when `MathML.parse_str()` is called on the line `"<ci><mml:msub><mml:mi>infectious</mml:mi><mml:mi>population</mml:mi></mml:msub></ci>"` 

I'm not sure if it's a malformed xml statement or if the parser is just being dumb

**Comment by fivegrant** [comment]
I went ahead and peaked at the dashboard. Here are the results given the current state of the branch @sabinala @jClugstor 

### CIEMSS Service
image

### SciML Service
image

File: scenarios/seir-stockflow/sciml/calibrate.json
Comment (fivegrant) [reply scenarios/seir-stockflow/sciml/calibrate.json:1]: thanks for opening https://github.com/DARPA-ASKEM/simulation-integration/issues/46!

**Comment by fivegrant** [comment]
Look at `docker compose logs sciml-service -f`. If you run this before `docker compose run tests`, you'll also be able to see them live.

File: scenarios/seir-stockflow/pyciemss/optimize-calibrate.json
Comment (sabinala) [reply scenarios/seir-stockflow/pyciemss/optimize-calibrate.json]: Just fixed this as well.

File: scenarios/seir-stockflow/sciml/calibrate.json
Comment (fivegrant) [reply scenarios/seir-stockflow/sciml/calibrate.json:1]: resolving for now since this is in a separate issue

**Review by fivegrant — APPROVED** [review]
looks good now

**Pull request merged**
